revolv/administrator/views.py

```diff
@@ -29,6 +35,423 @@ def get_context_data(self, **kwargs):
         return context
 
 
+class AdministratorAccountingView(UserDataMixin, TemplateView):
+    """View of the administrator accounting page.
+    """
+    template_name = 'administrator/accounting.html'
+
+    def clean_project_name(self, name):
```

Comment:
This could use a docstring...but that replace is kind of ugly so I be there might be a better way to do it haha